its/plugin/tests/src/test/java/com/sonar/it/java/suite/SonarLintTest.java

```diff
@@ -124,7 +124,6 @@ public void simpleTestFileJava() throws Exception {
     sonarlintEngine.analyze(standaloneAnalysisConfiguration, issues::add, null, null);
 
     assertThat(issues).extracting("ruleKey", "startLine", "inputFile.path", "severity").containsOnly(
-      tuple("java:S1607", 4, inputFile.getPath(), "MAJOR"),
```

Comment:
We should also explain why this issue is not raised anymore (requires semantic) in the code of the test (line 105).